File: FirstSDK/FirstSDKObjcChecker/ViewController.m
#import "ViewController.h"
@import FirstSDK;

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    [FirstSDKBase displayStaticLog];
    
    FirstSDKBase *firstSDK = [FirstSDKBase new];
    [firstSDK argumentChecker2WithMessage:@"ラベルあり引数の文字列"];
}


@end

File: FirstSDK/FirstSDKChecker/ViewController.swift
//
//  ViewController.swift
//  FirstSDKChecker
//
//  Created by funakoshi-dev on 2023/11/30.
//

import UIKit
import FirstSDK

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
        FirstSDKBase.displayStaticLog()
        FirstSDKBase().displayInstanceLog()
        FirstSDKBase().argumentChecker("**** Checker argument1")
        FirstSDKBase().argumentChecker2(message: "**** Checker argument2")
    }
}

